src/lib/Libnet/net_server.c

```diff
@@ -860,25 +820,26 @@ close_conn(int sd)
  * 	Given an index within the svr_conn array, reset all fields back to
  * 	their defaults and clear any select/poll related flags.
  *
- * @param[in]	cndx: index of the svr_conn entry
+ * @param[in]	indx: index of the svr_conn entry
  *
- * @return	void
  */
 static void
-cleanup_conn(int cndx)
+cleanup_conn(int indx)
```

Comment:
Why not use "idx" rather than "indx" to be consistent with other functions?

Reply:
Thanks for pointing out. Changing this to idx.

Reply:
Maybe it's GitHub, but it still looks like "indx" rather than "idx".

Reply:
Darn! It does look like i missed it again. I will recheck and update if necessary. Thanks for being vigilant.